lib/l10nUtil.js

```diff
@@ -44,6 +44,10 @@ const chromiumPrintingStringsPartPath = path.resolve(path.join(srcDir, 'chrome',
 const bravePrintingStringsPartPath = path.resolve(path.join(srcDir, 'brave', 'app', 'printing_strings.grdp'))
 const chromiumProfileSettingsPartPath = path.resolve(path.join(srcDir, 'chrome', 'app', 'profiles_strings.grdp'))
 const braveProfileSettingsPartPath = path.resolve(path.join(srcDir, 'brave', 'app', 'profiles_strings.grdp'))
+const chromiumFileManagerStringsPartPath = path.resolve(path.join(srcDir, 'chrome', 'app', 'file_manager_strings.grdp'))
+const braveFileManagerStringsPartPath = path.resolve(path.join(srcDir, 'brave', 'app', 'file_manager_strings.grdp'))
+const chromiumNuxStringsPartPath = path.resolve(path.join(srcDir, 'chrome', 'app', 'nux_strings.grdp'))
+const braveNuxStringsPartPath = path.resolve(path.join(srcDir, 'brave', 'app', 'nux_strings.grdp'))
```

Comment:
These need to be included since grdp files are newly added to the parent grd file which we process.